tests/integration/test_examples.py

```diff
@@ -86,3 +86,31 @@ def test_scan(scan_cleanup):
     """
     # Run entire scan.py script
     runpy.run_path("scan.py")
+
+
+@pytest.fixture
+def csv_cleanup(examples_as_cwd):
+    """Delete any files produced by csv_output.py.
+
+    :param examples_as_cwd: fixture to set examples dir as cwd
+    :type examples_as_cwd: None
+    """
+    yield
+
+    # Teardown: delete produced files
+    csv_files = Path.cwd().glob("*csv_output_*")
+    for file in csv_files:
+        if "_MFILE.DAT" not in file.name:
+            os.remove(file)
+
+
+def test_csv(csv_cleanup):
+    """Run the csv_output.py script and check no exceptions are raised.
+
+    csv_output.py intentionally produces files when running the notebook, but remove
+    them when testing.
+    :param csv_cleanup: fixture to delete any produced files
+    :type csv_cleanup: None
+    """
+    # Run entire scan.py script
+    runpy.run_path("csv_output.py")
```

Comment:
Can this perform an assertion on the .csv file contents, like if there are numbers in in?